### ISSUE TITLE
Version: 3.0.2166 Git 4feaa9ee - error C2220 in to phlib\mapimg.c(1885)

### ISSUE THREAD

**Issue by VictorVG** [processhacker/processhacker#377]
Version: 3.0.2166 (4feaa9ee) [master]

[00:00] Building ProcessHacker (x32)...
[ERROR]

phlib\mapimg.c(1885): error C2220: warning processed as an error - the file "object" was not created [phlib\phlib.vcxproj]
phlib\mapimg.c(1885): warning C4244: =: convert "ULONGLONG" to "ULONG_PTR", data loss is possible [phlib\phlib.vcxproj]

**Comment by VictorVG** [comment]
cl display error in to code fragment:

```
    {
        PULONGLONG array = (PULONGLONG)(PULONG_PTR)TlsCallbacks->CallbackAddress;

        for (i = 0; i < count; i++)
        {
            TlsCallbacks->Entries[i].Address = array[i];
        }
    }
```

code strung -  `TlsCallbacks->Entries[i].Address = array[i];`

**Comment by VictorVG** [comment]
Thanks! Fixed in to 7441ae2684528c9050e66
